react-components/src/index.js

```diff
@@ -0,0 +1,3 @@
+import './dev/CopyTextArea';
+import './dev/ToolDetails';
+import './dev/CategoryTree';
```

Comment:
This is the dev-mode entry point (that populates the `public/dev/index.html` file in `npm start` dev mode), and the `lib.js` file is the entry point for the DE.